docs/src/pages/configurations/options-parameter/index.md

```diff
@@ -5,68 +5,90 @@ title: 'Options Parameter'
 
 Storybook UI is configurable using an options API that allows you to tweak its appearance globally and for each story.
 
-> NOTE: If you've used older versions of Storybook this is formerly [addon-options](https://github.com/storybookjs/storybook/tree/next/addons/options), which has been deprecated.
-
 ### Global options
 
-Import and use `addParameters` with the `options` key in your `config.js` file.
+Import and use `setConfig` in your `manager.js` file.
 
 ```js
 import { addons } from '@storybook/addons';
 
 addons.setConfig({
+  /**
+   * show story component as full screen
+   * @type {Boolean}
+   */
+  isFullscreen: false,
+
   /**
    * display panel that shows a list of stories
    * @type {Boolean}
    */
   showNav: true,
+
   /**
    * display panel that shows addon configurations
    * @type {Boolean}
    */
   showPanel: true,
+
   /**
    * where to show the addon panel
    * @type {('bottom'|'right')}
    */
   panelPosition: 'bottom',
+
   /**
    * display the top-level grouping as a "root" in the sidebar
    * @type {Boolean}
    */
   showRoots: false,
+
   /**
    * sidebar tree animations
    * @type {Boolean}
    */
   sidebarAnimations: true,
+
   /**
    * enable/disable shortcuts
    * @type {Boolean}
    */
   enableShortcuts: true,
+
   /**
    * show/hide tool bar
    * @type {Boolean}
    */
   isToolshown: true,
+
   /**
    * theme storybook, see link below
    */
   theme: undefined,
-  /**	
-   * show story component as full screen	
-   * @type {Boolean}	
-   */	
-  goFullScreen: false,	
+
   /**	
    * id to select an addon panel	
    * @type {String}	
    */	
-  selectedPanel: undefined, // The order of addons in the "Addon panel" is the same as you import them in 'addons.js'. The first panel will be opened by default as you run Storybook	
+  selectedPanel: undefined,
 });
 ```
 
+### Sorting stories
+
+Import and use `addParameters` with the `options` key in your `preview.js` file.
+
+```js
+import { addParameters, configure } from '@storybook/react';
```

Comment:
Maybe we start using `@storybook/client-api` now?

Reply:
That sounds like a cool next step indeed!